app/components/flash_component.rb

```diff
@@ -2,6 +2,7 @@
 
 class FlashComponent < ViewComponent::Base
   TYPE_THEMES = {
+    success: 'success',
```

Comment:
lol, while I generally agree that this should probably be success across the board rather than notice, I'm wondering whether this is the PR to do it or whether we can use the existing `notice` Flash and kick updating to `success` to another PR?